ovos_dinkum_listener/voice_loop/voice_loop.py

```
@@ -711,15 +712,15 @@
 
         # get text and trigger callback
         try:
-            utts = self.stt.transcribe() or []
+            utts = self.stt.transcribe(lang=lang) or []
         except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)